fsw/mcp750-vxworks/src/cfe_psp_memory.c

```diff
@@ -59,7 +59,9 @@
 /*
 ** Define the cFE Core loadable module name
 */
-#define CFE_MODULE_NAME "cfe-core.o"
+#define CFE_MODULE_NAME_DEFAULT "cfe-core.o"
```

Comment:
Would it make sense to relate this to the build definition vs redefining here, or maybe an #ifndef?  Feels like a duplicate definition which could be single sourced (see EXE_TARGET).

Reply:
Yes, but I did notice that there is a separate issue about that (see #111) and so I figured it would be better to keep this pull request isolated to this specific warning, then fix the hardcoded definition of `cfe-core.o` in a pull request associated with the other ticket.